### PR TITLE
1341 add strict check of parameters in test properties

### PR DESCRIPTION
**Pull request by KseniiaSemenova94**

### CODE REVIEW

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light/src/test/java/io/github/epam/properties/PageLoadStrategyTests.java  2
- jdi-light/src/test/java/io/github/epam/properties/BrowserSizeTests.java  2
- jdi-light/src/test/java/io/github/epam/properties/PageCheckAfterOpenTests.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4576689&bid=15378013)

File: jdi-light/src/main/java/com/epam/jdi/light/common/Property.java [hunk 1]
```diff
@@ -3,7 +3,14 @@
 public enum Property {
 
     TIMEOUT_WAIT_ELEMENT("timeout.wait.element", "^[1-9][0-9]{1,2}$|^\\d$",
-            "Value of 'timeout.wait.element' must be an integer from 0 to 999.");
+            "Value of 'timeout.wait.element' must be an integer from 0 to 999."),
+    BROWSER_SIZE("browser.size", "MAXIMIZE|^([1-9][0-9]{2,3})x([1-9][0-9]{2,3})$",
```
Comment (IgorGaidukEpam) [inline jdi-light/src/main/java/com/epam/jdi/light/common/Property.java]: Great job!

File: jdi-light/src/test/java/io/github/epam/properties/BrowserSizeTests.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    @Test(dataProvider = "negativeData")
+    public void negativeTest(String value) {
+        Properties properties = new Properties();
+        properties.setProperty(BROWSER_SIZE.getName(), value);
+        try {
+            validateProperties(properties);
+            fail("Value '" + value + "' should not be valid for this test.");
+        } catch (InvalidParameterException exp) {
+            String expMessage = exp.getMessage();
+            assertEquals(expMessage,BROWSER_SIZE.getExMsg() + " See example: https://jdi-docs.github.io/jdi-light/#driver-settings");
```
Comment (IgorGaidukEpam) [inline jdi-light/src/test/java/io/github/epam/properties/BrowserSizeTests.java]: Add this
public static final String EXAMPLE_MESSAGE = " See example: https://jdi-docs.github.io/jdi-light/#driver-settings";
to the PropertyValidationUtils.java
and import it here and in another 2 places of your code.

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light/src/test/java/io/github/epam/properties/BrowserSizeTests.java  2
- jdi-light/src/test/java/io/github/epam/properties/PageLoadStrategyTests.java  2
- jdi-light/src/test/java/io/github/epam/properties/PageCheckAfterOpenTests.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4576689&bid=15378013)

File: jdi-light/src/main/java/com/epam/jdi/light/common/Property.java (hunk 1 above)
Comment (KseniiaSemenova94) [reply jdi-light/src/main/java/com/epam/jdi/light/common/Property.java]: Thanks a lot :)

File: jdi-light/src/test/java/io/github/epam/properties/BrowserSizeTests.java (hunk 2 above)
Comment (KseniiaSemenova94) [reply jdi-light/src/test/java/io/github/epam/properties/BrowserSizeTests.java]: I did it, but the same problem remains in the solutions of my colleagues.

**Pull request merged**